content/manager/bootstrapping.md

```diff
@@ -41,31 +45,65 @@ Please verify the [prerequisites]({{< relref "manager/bootstrapping.md" >}}) bef
 
 If you installed Cloudify using one of the premade packages, the manager blueprints should already be available to you.
 
-FILL HERE!
+* On Linux under /opt/cfy/cloudify-manager-blueprints
+* On Windows under c:\Program Files (x86)\Cloudify\cloudify-manager-blueprints-commercial
 
-If you didn't install using premade packages, clone the [cloudify-manager-blueprints](https://github.com/cloudify-cosmo/cloudify-manager-blueprints) repository from GitHub.
+If you didn't install using premade packages, download and extract the [cloudify-manager-blueprints](https://github.com/cloudify-cosmo/cloudify-manager-blueprints/archive/3.3.1.tar.gz) repository from GitHub.
 
 {{< gsHighlight  bash  >}}
 $ mkdir -p ~/cloudify-manager
 $ cd ~/cloudify-manager
-$ git clone https://github.com/cloudify-cosmo/cloudify-manager-blueprints
-$ cd cloudify-manager-blueprints
-$ git checkout -b cloudify <tag>
+$ curl -L https://github.com/cloudify-cosmo/cloudify-manager-blueprints/archive/3.3.1.tar.gz -o cloudify-manager-blueprints.tar.gz
+$ tar -xzvf cloudify-manager-blueprints.tar.gz
+$ cd cloudify-manager-blueprints-3.3.1
+$ ls -l
 ...
```

Comment:
I'm not willing to have people install git to bootstrap. it is ridiculous. I'll have them download a tar/zip depending on the system